File: day006/src/test/java/com/pawelpluta/day006/PersonFixture.java
package com.pawelpluta.day006;

import com.pawelpluta.day006.Person;

import static org.apache.commons.lang3.RandomStringUtils.randomAlphabetic;

class PersonFixture {

    static String randomLastName() {
        return "lastName" + randomAlphabetic(10);
    }

    static Person personWithLastName(String lastName) {
        return new Person("firstName", lastName);
    }
}

File: day012/src/test/java/com/pawelpluta/day012/PersonFixture.kt
package com.pawelpluta.day012

import com.pawelpluta.day012.AddressFixture.any
import org.apache.commons.lang3.RandomStringUtils
import java.util.*
import kotlin.random.Random

internal object PersonFixture {
    fun Person.Companion.any(): Person {
        return Person(
                "firstName_${RandomStringUtils.randomAlphabetic(8)}",
                "firstName_${RandomStringUtils.randomAlphabetic(8)}",
                Random.nextInt(1970, 2010),
                Address.any(),
                null,
                emptyList())
    }
}